cmd/revokeuser.go

```diff
@@ -110,22 +117,25 @@ func (p *RevokeUserParams) Load(ctx ActionCtx) error {
 		return err
 	}
 
-	if p.user == "" {
+	if p.user == "" && p.userPubKey == "" {
 		n := ctx.StoreCtx().DefaultUser(p.AccountContextParams.Name)
 		if n != nil {
 			p.user = *n
 		}
 	}
 
-	if p.user == "" {
-		return fmt.Errorf("user is required")
+	if p.userPubKey == "" && p.user == "" {
+		return fmt.Errorf("user or user-public-key are required")
+	} else if p.userPubKey != "" && p.user != "" {
```

Comment:
this won't happen with the above...

Reply:
it will, if the above does not set a user